TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift

```diff
@@ -72,8 +79,8 @@ class AudioRecordingsTableViewController: UITableViewController {
                 let newFileName = editAlertController.textFields?[0].text
                 
                 if let newFileName = newFileName {
-                    let audioCell = UITableViewCell(style: .default, reuseIdentifier: "audio")
-                    audioCell.textLabel?.text = newFileName
+                    let audioCell = tableView.dequeueReusableCell(withIdentifier: "audio", for: indexPath) as! AudioRecordingCell
```

Comment:
You don't need to do this if you call reloadData

Reply:
Where would I call reloadData? After I initialize the cell?

Reply:
No. You are already calling reloadData after the fileName has been updated.
Add a note here so I can explain the table view again so I'm sure you understand what's happening :)

Reply:
So your dataSource needs to drive the tableView, not the other way around.